dlt/normalize/normalize.py

```diff
@@ -211,7 +248,8 @@ def _gather_metrics_and_close(
                     )
                     root_tables.add(root_table_name)
                     normalizer = _get_items_normalizer(
-                        DataWriter.item_format_from_file_extension(parsed_file_name.file_format)
+                        DataWriter.item_format_from_file_extension(parsed_file_name.file_format),
+                        stored_schema["tables"].get(root_table_name),
```

Comment:
you do not handle `table_schema` being `None` in `_get_items_normalizer` - and this is a normal case (and we have failing low level tests IMO due to this)

Reply:
Added handling for `table_schema is None`.